server/services/locationService.test.ts

```diff
@@ -0,0 +1,79 @@
+import serviceCreator from './locationService'
+
+const elite2ClientBuilder = jest.fn()
+import incidentClient = require('../data/incidentClient')
+const token = 'token'
+
+const elite2Client = {
+  getPrisons: jest.fn(),
+  getPrisonById: jest.fn(),
+}
+
+incidentClient.updateAgencyId = jest.fn()
```

Comment:
if you overwrite this method like this, you do so for all other tests.
The correct way of mocking requires is to use jest.mock but the standard pattern we use is to inject it as a dependency.

(Could argue that it is surprising that the update of prison is on here - If I was looking for it, I would expect it to be on the report service)

Reply:
done